cmake/helpers/CheckDependentLibraries.cmake

```diff
@@ -298,15 +276,10 @@ option(GDAL_USE_LIBLERCV1_INTERNAL "Set ON to build mrf driver with internal lib
 
 # DISABLED_BY_DEFAULT since it prevents MRF Lerc support (see frmts/mrf/CMakeLists.txt)
 gdal_check_package(LERC "Enable LERC (external)" CAN_DISABLE DISABLED_BY_DEFAULT)
-if (NOT GDAL_USE_LERC)
-  set(GDAL_USE_LIBLERC_INTERNAL
-      ON
-      CACHE BOOL "Use internal liblerc copy (if set to ON, has precedence over GDAL_USE_LERC)")
-else ()
-  set(GDAL_USE_LIBLERC_INTERNAL
-      OFF
-      CACHE BOOL "Use internal liblerc copy (if set to ON, has precedence over GDAL_USE_LERC)")
-endif ()
+invert_on_off(GDAL_USE_LERC NOT_GDAL_USE_LERC)
+set(GDAL_USE_LIBLERC_INTERNAL
```

Comment:
I would prefer if the variable was named `GDAL_USE_LERC_INTERNAL`, similar to `GDAL_USE_LERC`.

Reply:
done